setup.cfg

```diff
@@ -127,10 +127,7 @@ show_error_codes = True
 [mypy-pandas.tests.*]
 check_untyped_defs=False
 
-[mypy-pandas.conftest]
-ignore_errors=True
-
-[mypy-pandas.tests.tools.test_to_datetime]
+[mypy-pandas.conftest,pandas.tests.window.conftest]
```

Comment:
similar error in pandas.tests.window.conftest as in pandas.conftest,pandas, so bundled together. 

this is a mypy inference issue we have encountered before

error: List item 0 has incompatible type "ParameterSet"; expected "Sequence[Collection[object]]"  [list-item]